homeassistant/components/tesla/sensor.py

```diff
@@ -88,6 +88,16 @@ def unit_of_measurement(self):
                 self.current_value = round(
                     convert(self.current_value, LENGTH_MILES, LENGTH_KILOMETERS), 2
                 )
+        elif self.tesla_device.type == "charging rate sensor":
```

Comment:
Just a comment -- is there a constant in the library you can reference instead of the static string?

Reply:
Not at this point unfortunately.